tests/unit/test_pipeline_model.py

```diff
@@ -111,17 +111,58 @@ def test_deploy_endpoint_name(tfo, time, sagemaker_session):
     framework_model = DummyFrameworkModel(sagemaker_session)
     sparkml_model = SparkMLModel(model_data=MODEL_DATA_2, role=ROLE, sagemaker_session=sagemaker_session)
     model = PipelineModel(models=[framework_model, sparkml_model], role=ROLE, sagemaker_session=sagemaker_session)
-    model.deploy(endpoint_name='blah', instance_type=INSTANCE_TYPE, initial_instance_count=55)
+    model.deploy(instance_type=INSTANCE_TYPE, initial_instance_count=1)
     sagemaker_session.endpoint_from_production_variants.assert_called_with(
-        'blah',
+        'mi-1-2017-10-10-14-14-15',
```

Comment:
Where is this string coming from?

Reply:
Seems like it's 'model image 1' followed by a timestamp.

That's not actually part of my diff, though. I've only added test_transformer -- git's getting it wrong since the two methods start the same way.